packages/a/angular.js.json

```
@@ -18,13 +17,16 @@
     "type": "git",
     "url": "git://github.com/angular/angular.js.git"
   },
-  "npmName": "angular",
-  "npmFileMap": [
-    {
-      "basePath": "",
-      "files": [
-        "angular.*"
-      ]
-    }
-  ]
+  "autoupdate": {
+    "source": "npm",
+    "target": "angular",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "angular.*"
```

```suggestion
          "angular?(.min).js?(.map)"
```
Per @drewfreyling's comments, I think we just need the JS files + the map?

[Glob preview](https://www.digitalocean.com/community/tools/glob?comments=true&glob=angular%3F%28.min%29.js%3F%28.map%29&matches=false&tests=%2F%2F%20Imported%20from%20angular%20NPM%20package&tests=angular-csp.css&tests=angular.min.js.gzip&tests=angular.js&tests=angular.min.js&tests=index.js&tests=bower.json&tests=package.json&tests=angular.min.js.map&tests=LICENSE.md&tests=README.md)

---

Thanks! But I think we also need `angular-csp.css`